default/AI/ShipDesignAI.py

```diff
@@ -462,6 +468,8 @@ def _update_buildable_items_this_turn(self, verbose=False):
             if res:
                 if verbose:
                     print "Success: Added Test Design %s, with result %d" % (testdesign_name, res)
+                # update cache since we just created a new design in fo - dummy_id is not used
+                dummy_id = _get_design_by_name(testdesign_name,update_invalid=True).id
```

Comment:
If you want to indicate you do not use a return value, use an underscore as variable:
```_ = _get_design_by_name(testdesign_name,update_invalid=True)```

Note I also omitted the ```.id``` which we do not care about anyway (and may throw an error, see below).